File: src/Homeworks/Homework4/Calculator.fs
module Homework4.Calculator

let calculate (val1: int) operation (val2:int) =
        match operation with
        |"+" -> val1 + val2
        |"-" -> val1 - val2
        |"*" -> val1 * val2
        |":" -> val1 / val2
        |"/" -> val1 / val2
        | _ -> 0
       

File: src/Homeworks/Homework1/Calculator.cs
namespace Homework1
{
	public static class Calculator
	{
		public static int Calculate(int val1, string operation, int val2)
		{
			var result = operation switch
			{
				"+" => val1 + val2,
				"-" => val1 - val2,
				"*" => val1 * val2,
				":" => val1 / val2,
				"/" => val1 / val2,
				_ => 0
			};
			return result;
		}
	}
}